app/components/icons/ArrowFilterIcon.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from "react";
+
+export default function ArrowFilterIcon({ classNames = "h-6 w-6" }) {
+  const [isActive, setIsActive] = useState(false);
+
+  const [isHover, setIsHover] = useState(false);
+
+  const handleMouseEnter = () => {
+    setIsHover(true);
+  };
+
+  const handleMouseLeave = () => {
+    setIsHover(false);
+  };
+
+  const arrowStyle = {
+    color: isHover ? "#9cc317" : "",
+  };
```

Comment:
I'm not sure if we need all this logic here, or it can happen on the parent component by putting everything inside of a 'button' tag, styling the hover there and using the default `coursor:pointer`

The green color should happen on hover over every clickable element in this filter 'button' (in this case the filter icon, the label and the arrow), right now it only happens on the arrow, although all the elements perform the same action. In that case, the button solution mentioned above might help

Reply:
![Uploading Screenshot 2022-11-03 at 13.26.40.png…]()
Some elements on the navbar are also implemented as 'buttons', so on hover and on click, it all acts together (in this case, both the world icon and the DE text has the same behavior), although you don't see any button related style (no border etc)

Reply:
I‘d also leave the hover interaction to CSS (hover:text-vsp-500 as an example, not sure about the correct color 😅) instead of relying on JavaScript here 😊